### PR TITLE
Fibonacci Series(Dp)

### PR DESCRIPTION
**Pull request by ashishsahu1**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by ashishsahu1** [comment]
Screenshot1
Screenshot2

Outputs for both is some thing like this

**Review by siddharth25pandey — APPROVED** [review]
@ashishsahu1 Looks fine to me
Able to merge
 Thanks!

File: Python/dp/fibinacci(TopDown).py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    c=int(input('Enter your choice : '))
+    if c==1:
+        n=int(input('Enter How many terms you want to print : '))
+        for i in range(n):
+            print(dynamicFibonacci(i),end=" -> ")
+    elif c==2:
+        n=int(input('Enter the term for which you want to get fibonacci value  : '))
+        print(dynamicFibonacci(n))
+    else:
+        print('Wrong choice')
```
Comment (Kajol-Kumari) [inline Python/dp/fibinacci(TopDown).py]: please add one extra blank line at the end of the file.

File: Python/dp/fibonacci(BottumUp).py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        n=int(input('Enter How many terms you want to print : '))
+        for i in range(n):
+            print(fibonacciBottomUp(i),end=" -> ")
+    elif c==2:
+        n=int(input('Enter the term for which you want to get fibonacci value  : '))
+        print(fibonacciBottomUp(n))
+    else:
+        print('Wrong choice')
+
+
```
Comment (Kajol-Kumari) [inline Python/dp/fibonacci(BottumUp).py]: please remove the unnecessary blank lines left here.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@ashishsahu1 Minor changes required, rest Looks good to me :)

**Comment by ashishsahu1** [comment]
@Kajol-Kumari I have done with those changes.. can you review again

**Pull request closed without merging**